neptune/new/exceptions.py

```diff
@@ -155,6 +155,29 @@ def __init__(self, run_uuid: uuid.UUID):
         super().__init__("Run with UUID {} not found. Could be deleted.".format(run_uuid))
 
 
+class InactiveRunException(NeptuneException):
+    def __init__(self, short_id: str):
+        message = """
+{h1}
+----InactiveRunException----------------------------------------
```

Comment:
@Herudaio could you confirm you like this help screen? :)

Reply:
Yes it's his version